ailab/db/__init__.py

```diff
@@ -33,7 +29,6 @@ def raise_error(message):
 
 def connect_db():
     """Connect to the postgresql database and return the connection."""
-    logger.info(f"Connecting to {LOUIS_DSN}")
     connection = psycopg.connect(
         conninfo=LOUIS_DSN,
```

Comment:
Should we change the secrets from [LOUIS_...] to [AI_LAB_...] if we want to remove all references?

Reply:
Good point. I would suggest to rewrite packages in a way that they don't force the env var naming on their users because it could lead to conflicts. For instance, if I already had `POSTGRES_URI` in my repo, `ailab-db` would force me to rename it to `LOUIS_DSN` or have a duplicate of the same value. What I suggest is to pass the value to the function that needs it.

Fixing this would even greatly improve testability. I suggest this be part of #92.